(DOCS-28243): Add Kotlin link to Custom Function Auth page (#366)
mongodb/docs-app-services: source/authentication/custom-function.txt

The function accepts any value provided from your client app, so the
actual field names and values depend on your implementation.

For examples of how to create a custom credential ``payload`` object

> using the Realm SDKs, refer to the documentation for a specific SDK:

Reviewer: ```suggestion
using the Realm SDKs, refer to the documentation for each SDK:
```

> - :ref:`C++ SDK (Alpha) <cpp-login-custom-function>`

Reviewer: ```suggestion
- :ref:`C++ SDK <cpp-login-custom-function>`
```

nit more maintainable to not include "Alpha" here for when it's updated to beta, rc, ga, etc

Author: Good catch, thanks!
